### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by drupol**
This pr relates to #452 .

### CODE REVIEW

**Comment by drupol** [comment]
@ciaranmcnulty Do you have any feedback on this? Should we keep tests with php < 5.6 ? Is it still required ?

**Comment by drupol** [comment]
As PHPSpec 3 is in end of life, shouldn't we also update its version ?

File: spec/Prophecy/Promise/ThrowPromiseSpec.php [hunk 1]
```diff
@@ -48,9 +48,9 @@ function it_throws_error_instances(ObjectProphecy $object, MethodProphecy $metho
             throw new SkippingException('The class Error, introduced in PHP 7, does not exist');
         }
 
-        $this->beConstructedWith($exc = new \Error('Error exception'));
+        $this->beConstructedWith(new \Error('Error exception'));
 
-        $this->shouldThrow($exc)->duringExecute(array(), $object, $method);
+        $this->shouldThrow(\Error::class)->duringExecute(array(), $object, $method);
```
Comment (ciaranmcnulty) [inline spec/Prophecy/Promise/ThrowPromiseSpec.php]: Why is this changed?

Comment (drupol) [reply spec/Prophecy/Promise/ThrowPromiseSpec.php]: I think this was unrelated, I reverted it.

**Comment by ciaranmcnulty** [comment]
@drupol we can't update past phpspec 3 apparently because of BC breaks

**Comment by drupol** [comment]
@ciaranmcnulty as long as we support unsupported PHP version (< 5.6), then it's impossible to test with lowest deps on Travis.
I will revisit this as soon as we will drop those from Travis.
